plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduIntegrationDecimalColumns.java

```diff
@@ -144,7 +131,7 @@ private void doTestCreateTableWithDecimalColumn(TestDecimal decimal)
 
         String getTableName()
         {
-            return format("test_decimal_%s_%s", precision, scale);
+            return format("test_decimal_%s_%s_%s", randomTableSuffix(), precision, scale);
```

Comment:
Isn't this redundant now? `TestTable` will add yet another suffix

Reply:
Yes !! Missed it